camera1-dist.conf.in

```diff
@@ -1,11 +1,19 @@
-# @sysconfdir@/motion/thread1.conf
+# @sysconfdir@/motion/camera1.conf
 #
 # This config file was generated by @PACKAGE_NAME@ @PACKAGE_VERSION@
 
 ###########################################################
 # Capture device options
 ############################################################
 
+#
+# Camera Id
+# Id used to label the camera when inserting data into SQL or saving the
+# camera image to disk.  This is better than using thread number so that there
+# always is a consistent labe
```

Comment:
We probably don't need the last sentence here anymore, as thread number is hopefully no longer too visible to the user. (same comment applies to other 3 config files too!)